samples/animated_gradient.py

```diff
@@ -17,29 +17,32 @@ def get_gradation_2d(start, stop, width, height, is_horizontal):
 
 
 def get_gradation_3d(width, height, start_list, stop_list, is_horizontal_list):
-    result = np.zeros((height, width, len(start_list)), dtype=np.float)
+    result = np.zeros((height, width, len(start_list)), np.float)
 
     for i, (start, stop, is_horizontal) in enumerate(zip(start_list, stop_list, is_horizontal_list)):
         result[:, :, i] = get_gradation_2d(start, stop, width, height, is_horizontal)
 
     return result
 
 
-with pyvirtualcam.Camera(width_, height_, fps_, print_fps=True) as cam:
-    print(f'Virtual cam started ({width_}x{height_} @ {fps_}fps)')
-    j = 0
+with pyvirtualcam.Camera(WIDTH, HEIGHT, FPS, print_fps=True) as cam:
```

Comment:
Maybe for consistency with the other example it would be better to just inline the values like `width=1280, height=720, fps=20` and then reference them with `cam.width` etc. What do you think?

Reply:
nice